Dockerfile

```
@@ -38,12 +38,13 @@ RUN ./bootstrap \
 
 ## libcoap
 RUN git clone --recursive -b 'develop' \
-    'https://github.com/obgm/libcoap.git' /tmp/libcoap
+    'https://github.com/obgm/libcoap.git' /tmp/libcoap && \
+    cd /tmp/libcoap && git status && \
```

It works fine without `git status`
```suggestion
    cd /tmp/libcoap && \
```